docs/src/main/paradox/user/lang/dart/serialization.md

```diff
@@ -0,0 +1,5 @@
+# Serialization
+
+Cloudstate functions serve gRPC interfaces, and naturally the input messages and output messages are protobuf messages that get serialized to the protobuf wire format. However, in addition to these messages, there are a number of places where Cloudstate needs to serialize other objects, for persistence and replication.
+
+See [here](/docs/src/main/paradox/user/lang/java/serialization.md) for more details
```

Comment:
is this the correct link to the java serialization? The link also does not work.

Perhaps it should point to the generic serialization page located here?
https://cloudstate.io/docs/developer/language-support/serialization.html#json-values

BTW. The same Link in the Kotlin Version also does not work and points to:
https://cloudstate.io/docs/src/main/paradox/user/lang/java/serialization.md

Reply:
Good